budget/test/test_duplicate_budget.yml

```diff
@@ -4,17 +4,43 @@
   !context
     uid: 'res_users_beth'
 -
+  I first create a budget item
+-
+  !record {model: budget.item, id: budget_item_1}:
```

Comment:
as I know with this issue https://github.com/odoo/odoo/issues/8107, it will be done with 'admin' account ...